### PR TITLE
Update total-rewards.md

### PR DESCRIPTION
**Pull request by iogungbade**

### CODE REVIEW

**Comment by iogungbade** [comment]
I added a section on "how we determine compensation".

**Review by natalie-hub — CHANGES_REQUESTED** [review]
I reviewed feedback/edits with @iogungbade directly, and she'll make changes to this PR.

**Review by natalie-hub — DISMISSED** [review]
Looks good, sorry for delay

File: operations/workplace/people/total-rewards.md [hunk 1]
```diff
@@ -21,6 +21,48 @@ To learn more about how the total rewards guiding principles drive equitable and
...
+
+**Role and Level**
+
+Our compensation system aims to position salary for each role and level, at or around the median of Mattermost’s labour market, which comprises companies in the technology industry that are similar in terms of size and growth stage.
+
+**Local Market Compensation** 
+
+We pay competitive local market rates. This means that our compensation system targets pay at or around the median salaries of comparable organizations in each local market. The local market compensation approach is based on an extensive internal analysis of the cost of labour (local market compensation) in over 12,000 cities across the globe, and it enables Mattermost to support equitable standard of living for staff members.
+
+- _**Why we align compenstion to the local market**_
```
Comment (cwarnermm) [inline operations/workplace/people/total-rewards.md]:
```suggestion
- _**Why we align compensation to the local market**_ 
```

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**